CDPBatchEditor/Commands/Command/Interface/IValueSetCommand.cs

```diff
@@ -35,5 +35,8 @@ public interface IValueSetCommand
         /// Move the value of reference value to manual value on value sets of parameters of specified element definition
         /// </summary>
         void MoveReferenceValuesToManualValues();
+        void MoveManualValuesToReferenceValues();
```

Comment:
Please add comments to these new interface methods as also done, for example, for MoveReferenceValuesToManualValues.